docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+If you have common configuration options you want to apply to all profiles, you have to do this manually per profile. See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration Options]
```

Comment:
One sentence per line, please. Also, a period is needed after the `xref` to Setting Global Configuration Options.